tf/azure/r.list

```diff
@@ -3,12 +3,12 @@ westus	UP
 eastus2	UP
 westus2	UP
 southcentralus	UP
-northcentralus	UP
-northeurope	UP
-westeurope	UP
-uksouth	UP
-centralus	UP
-eastasia	UP
-koreacentral	UP
-japaneast	UP
-canadacentral	UP
\ No newline at end of file
+northcentralus	DOWN
+northeurope	DOWN
```

Comment:
why all DOWN?

Reply:
these location are not init yet and also need test availability. i will switch them to UP after finish init.